tests/functional/test_read_configuration.py

```diff
@@ -55,3 +58,30 @@ def test_read_config(config_directory, config_file, expected):
     result = api.read_configuration(config_filename)
 
     assert result == expected
+
+
+@pytest.mark.parametrize(
+    "default_encoding",
+    (
+        "cp1252",
+        "UTF-8",
+    ),
+)
+def test_read_config_unicode(default_encoding, monkeypatch):
```

Comment:
This could do with an explanatory comment explaining what you told me in the pull request description 😄 .

Reply:
Will add 👍